console/atest-ui/src/views/net.ts

```diff
@@ -352,14 +336,13 @@ function GenerateCode(request: GenerateRequest,
       Generator: request.generator
     })
   }
-  fetch('/server.Runner/GenerateCode', requestOptions)
+  fetch(`/api/v1/code-generators/generate`, requestOptions)
```

Comment:
IMO, here we can all use camel-named hair to keep the format of the uri consistent

`\xxx\codeGenerators\xxx`